src/Pact/Types/Gas.hs

```diff
@@ -156,7 +156,7 @@ instance Pretty GasArgs where
     GConcatenation i j -> "GConcatenation:" <> pretty i <> colon <> pretty j
     GUnreduced {} -> "GUnreduced"
     GPostRead rv -> "GPostRead:" <> pretty rv
-    GPreWrite wv -> "GWrite:" <> pretty wv
+    GPreWrite wv _ -> "GWrite:" <> pretty wv
```

Comment:
why elide the argument?

Reply:
Sure I can include it, but there was pretty instance for `SizeOfVersion`.